components/KitButton.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Image, Text, TouchableOpacity } from 'react-native';
+import KitText from './KitText';
+/*
+props expected:
+  style: {
+    button:{
+
+    },
+    image: {
+
+    }
+  }
+  
+  onPress
+  image
+
+  buttonBackgroundColor
```

Comment:
Maybe a future thing but I'd prefer having fixed button colors, ex: primary, secondary and using constants for colors

Reply:
I like the idea of using constants for colors, I'll go ahead and build a document for that now.

We will have to talk to design about what they have in mind for colors, wanted to keep the flexibility in the current implementation in case there was a redesign